app/views/reports/regions/_search_script.html.erb

```diff
@@ -0,0 +1,16 @@
+<script>
```

Comment:
Since this is all static, I would suggest moving this javascript to a js file.

Reply:
Good suggestion — we'll resolve this in a future PR when we overhaul our assets pipeline to webpack.